appveyor.yml

```diff
@@ -49,4 +49,4 @@ build_script:
 test_script:
   - usr\bin\julia -e "versioninfo()"
   - copy usr\lib\julia\sys.ji local.ji && usr\bin\julia -J local.ji -e "true" && del local.ji
-  - cd test && ..\usr\bin\julia runtests.jl all
+  - cd test && ..\usr\bin\julia runtests.jl libgit2 repl pkg
```

Comment:
Not sure if `all pkg` will work correctly, but if it doesn't let's add a separate line that just runs `pkg`. It takes less than 2 minutes on appveyor and the queue has been fine lately so let's add it.